nullaway/src/test/java/com/uber/nullaway/NullAwayJSpecifyArrayTests.java

```diff
@@ -110,6 +110,27 @@ public void arrayDeclarationAnnotation() {
         .doTest();
   }
 
+  @Test
+  public void arrayContentsAndTopLevelAnnotation() {
+    makeHelper()
+        .addSourceLines(
+            "Test.java",
+            "package com.uber;",
+            "import org.jspecify.annotations.Nullable;",
+            "class Test {",
+            "  static @Nullable String [] fizz = {\"1\"};",
+            "  static Object foo = new Object();",
+            "  static void foo() {",
+            "     if (fizz != null) {",
+            "       String s = fizz[0];",
+            "      // TODO: This should report due to dereference of @Nullable s",
```

Comment:
The failure here has nothing to do with annotations being in both places.  It's because we haven't updated our dataflow support yet.